### PR TITLE
Update LibAFL

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

**Comment by tokatoka** [comment]
update libafl to fix https://github.com/google/fuzzbench/pull/2028#issuecomment-2299979757 this bug.
@DonggeLiu on my local fuzzbench run libxml2 does not crash anymore can you run a experiment to test?

**Comment by tokatoka** [comment]
wait i'll just add this change to addison's branch, too

**Comment by DonggeLiu** [comment]
To maximize parallelization and minimize our waiting time, I will launch another experiment here for you to cross-check and debug/confirm the cause.

**Comment by DonggeLiu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name 2024-08-22-2036-bases-1 --fuzzers libafl

**Comment by tokatoka** [comment]
it didn't work. unlike the other two this one doesn't have experiment-folder even.
although the build log says everthing was fine

**Comment by DonggeLiu** [comment]
Experiment `2024-08-22-2036-bases-1` data and results will be available later at:
[The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=2024-08-22-2036-bases-1/).
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/2024-08-22-2036-bases-1/index.html).
[The experiment report(experimental)](https://storage.googleapis.com/www.fuzzbench.com/reports/experimental/2024-08-22-2036-bases-1/index.html).

**Comment by DonggeLiu** [comment]
> it didn't work. unlike the other two this one doesn't have experiment-folder even.

This is due to a [separate issue](https://pantheon.corp.google.com/logs/query;query=jsonPayload.experiment%3D2024-08-22-2036-bases-1;cursorTimestamp=2024-08-22T05:17:56.566518340Z;duration=P1D?project=fuzzbench&e=-13802955&mods=logs_tg_prod) from DB.

I will restart it

**Comment by DonggeLiu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name 2024-08-23-2036-bases-1 --fuzzers libafl

**Comment by DonggeLiu** [comment]
Experiment `2024-08-23-2036-bases-1` data and results will be available later at:
[The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=2024-08-23-2036-bases-1/).
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/2024-08-23-2036-bases-1/index.html).
[The experiment report(experimental)](https://storage.googleapis.com/www.fuzzbench.com/reports/experimental/2024-08-23-2036-bases-1/index.html).

**Comment by tokatoka** [comment]
Can we merge this?
The experiment looks good

**Review by DonggeLiu — APPROVED** [review]

File: service/gcbrun_experiment.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 TRIGGER_COMMAND = '/gcbrun'
 RUN_EXPERIMENT_COMMAND_STR = f'{TRIGGER_COMMAND} run_experiment.py '
 SKIP_COMMAND_STR = f'{TRIGGER_COMMAND} skip'
-# A DUMMY COMMENT
+# A DUMMY CoMMENT
```
[the remark this answers is not in the document]
Comment (tokatoka) [reply service/gcbrun_experiment.py]: done

[the remark this answers is not in the document]
Comment (DonggeLiu) [reply service/gcbrun_experiment.py]: Thanks!

**Pull request merged**